spring-boot-docs/src/main/asciidoc/appendix-application-properties.adoc

```diff
@@ -986,6 +986,7 @@ content into your application; rather pick only the properties that you need.
 	spring.kafka.listener.ack-time= # Time in milliseconds between offset commits when ackMode is "TIME" or "COUNT_TIME".
 	spring.kafka.listener.concurrency= # Number of threads to run in the listener containers.
 	spring.kafka.listener.poll-timeout= # Timeout in milliseconds to use when polling the consumer.
+	spring.kafka.listener.batch-listener= # If true listener container factory will be configured to create batch listeners.
```

Comment:
Documentation here matches exactly the field javadoc. If you had to change it here, it is a good indicator that the one in the metadata can be improved as well.